R/heatmap_mutation_frequency_bin.R

```diff
@@ -50,20 +50,25 @@
 #' @export
 #'
 #' @examples
-#' #load metadata.
-#' metadata = GAMBLR.data::gambl_metadata
-#' dlbcl_bl_meta = dplyr::filter(metadata, pathology %in% c("DLBCL", "BL"))
-#'
-#' #bring together all derived sample-level results from many GAMBL pipelines.
-#' dlbcl_bl_meta = collate_results(join_with_full_metadata = TRUE,
-#'                                 these_samples_metadata = dlbcl_bl_meta)
-#'
-#' #get ashm regions
-#' some_regions = GAMBLR.data::grch37_ashm_regions
-#'
-#' mut_count_matrix <- calc_mutation_frequency_bin_by_regions(
-#'    these_samples_metadata = dlbcl_bl_meta,
-#'    regions_bed = some_regions
+#' library(GAMBLR.data)
+#' library(dplyr)
+#' 
+#' # get meta data
+#' my_meta <- get_gambl_metadata() %>% 
+#'   filter(sample_id %in% c("DOHH-2", "OCI-Ly10", "OCI-Ly3", "SU-DHL-10", "SU-DHL-4"))
+#' 
+#' # get ashm regions of a set of genes.
+#' my_regions = GAMBLR.data::somatic_hypermutation_locations_GRCh37_v_latest %>%
+#'   rename( "chrom"="chr_name", "start"="hg19_start", "end"="hg19_end", "name"="gene") %>%
+#'   mutate( chrom = stringr::str_remove(chrom, "chr") )
+#' 
+#' # create heatmap of mutation counts for the specified regions
+#' meta_columns <- c("pathology", "lymphgen", "COO_consensus", "DHITsig_consensus")
```

Comment:
Just want to double-check that this is a working example?

Reply:
See the plot figure in my message above.